core/server/data/migrations/versions/2.13/1-remove-empty-strings.js

```diff
@@ -0,0 +1,74 @@
+const Promise = require('bluebird');
+const common = require('../../../../lib/common');
+const schema = require('../../../schema');
+
+/*
+ * [{
+ *   tableName: 'posts',
+ *   columns: ['custom_excerpt', 'description', 'etc...']
+ * }]
+ * */
+const tablesToUpdate = Object.keys(schema.tables).reduce((tablesToUpdate, tableName) => {
+    const table = schema.tables[tableName];
+    const columns = Object.keys(table).filter(columnName => table[columnName].nullable);
+    if (!columns.length) {
+        return tablesToUpdate;
+    }
+    return tablesToUpdate.concat({
+        tableName,
+        columns
+    });
+}, []);
+
+const createReplace = (connection, from, to) => (tableName, columnName) => {
+    return connection.schema.hasTable(tableName)
+        .then((tableExists) => {
+            if (!tableExists) {
+                common.logging.warn(
+                    `Table ${tableName} does not exist`
+                );
+                return;
+            }
+            return connection.schema.hasColumn(tableName, columnName)
+                .then((columnExists) => {
+                    if (!columnExists) {
+                        common.logging.warn(
+                            `Table '${tableName}' does not have column '${columnName}'`
+                        );
+                        return;
+                    }
+
+                    common.logging.info(
+                        `Updating ${tableName}, setting '${from}' in ${columnName} to '${to}'`
+                    );
+
+                    return connection(tableName)
+                        .where(columnName, from)
+                        .update(columnName, to);
+                });
+        });
+};
+
+module.exports.up = ({connection}) => {
```

Comment:
DML migrations should use a transaction.
https://github.com/TryGhost/knex-migrator#transactions